api/app/signals/settings/__init__.py

```diff
@@ -2,4 +2,33 @@
 # Copyright (C) 2018 - 2021 Gemeente Amsterdam
 # Temporary fix to make the settings restructure work with current deployment settings. Can be
 # removed after the Ansible playbooks for deployment are updated with the production settings.
-from signals.settings.base import *  # noqa
+import os, sys # noqa
+
+from logs import get_configuration
+
+from .base import *  # noqa
+from .feature_flags import * # noqa
+
+try:
+    from .local import * # noqa
```

Comment:
Will this interfere with running tests locally?
If for some example I overwrite a setting that breaks tests I am not aware of that 🤔 
Perhaps this should be in the `else` of the if statement  for testing?

Reply:
In this format if test are running they are overriding settings in the local.py if they are both defined in the local.py and settings.py
If tests break because of settings in the local.py they are a local issue.